src/migration/1697545231300-AddInvitedByIdToFeature.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddInvitedByIdToFeature1697545231300
```

Comment:
Not a major issue but I'd appreciate one migration per PR. Makes it easy to track all the db changes of a PR

Reply:
Not something we've done in the past, although agree with the sentiment.
Possibly something we can start doing from now on :)